### PR TITLE
moved files and depthfirst

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,5 +21,6 @@
 + [treeIntersection](javascript/code-challenges/treeintersection/README.md)
 + [leftJoin](javascript/code-challenges/leftJoin/README.md)
 + [graph](javascript/code-challenges/graph/README.md)
-+ [breadthFirst](javascript/code-challenges/breadthFirst/README.md)
-+ [getEdge](javascript/code-challenges/getEdge/README.md)
++ [breadthFirst](javascript/code-challenges/graph/breadthFirst/README.md)
++ [getEdge](javascript/code-challenges/graph/getEdge/README.md)
++ [depthFirst](javascript/code-challenges/graph/depthFirst/README.md)
```
Comment (adrianhuebner) [inline README.md:26]: Once you have a chance to get to testing I will regrade for you!
